site/content/docs/5.3/components/orange-navbar.md

```diff
@@ -202,7 +202,7 @@ An additional navbar (with text or icon items) can be added on the right of the
           </li>
           <li class="nav-item">
             <a href="#" class="nav-link nav-icon">
-              <img src="/docs/{{<param docs_version>}}/assets/img/navbar-contact.png" width="25" height="25" role="img" alt="User" loading="lazy">
```

Comment:
Just a question here, wouldn't it make sense in a real usage in a website to have the name of the user in this `alt=""`? For instance, `alt="Julien Déramond"` or `alt="Julien Déramond avatar"`, or anything else. I mean, it is decorative, but knowing that there's an image of you (or your avatar) is still valuable information, maybe.

Reply:
@julien-deramond I remove the suppression of the role from this PR, since it is done in #2704 .

To answer your question, of course it depends on the context of the project, but here it is fake, so... See the migration guide which indicates the developers to pay attention to this label.